src/components/colorPicker/ColorPickerDialog.tsx

```diff
@@ -22,45 +70,17 @@ const KEYBOARD_HEIGHT = 216;
  * @extends: Dialog
  * @example: https://github.com/wix/react-native-ui-lib/blob/master/demo/src/screens/componentScreens/ColorPickerScreen.js
  */
-class ColorPickerDialog extends PureComponent {
+class ColorPickerDialog extends PureComponent<Props, State> {
   static displayName = 'ColorPickerDialog';
```

Comment:
ColorPickerDialog is an internal ColorPicker component. 
If you want its props (interface) to be included in ColorPicker docs you'll need to set its displayName to be `ColorPicker` like the main component.

Reply:
I didn't know that. Why is a simple `export` and `extends` not enough?

Reply:
Done